onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.cc

```diff
@@ -172,13 +164,14 @@ std::unordered_map<std::string, vector<const Node*>> GetAllQuantizedOpInputs(con
 
 Status ModelBuilder::RegisterInitializers() {
   // First pass to get all the stats of the initializers
-  auto initializer_size = initializers_.size();
+  const auto& initializer_tensors(GetInitializerTensors());
```

Comment:
>initializer_tensors [](start = 14, length = 19)

very minor nit: technically initializers can only be tensors, so the '_tensor' part is redundant.

Reply:
I have another vector called initializers on the line below, have to find another name for this